packages/button/src/ButtonBase.ts

```diff
@@ -32,6 +32,10 @@ import buttonStyles from './button-base.css.js';
 export class ButtonBase extends ObserveSlotText(LikeAnchor(Focusable), '', [
     'sp-overlay,sp-tooltip',
 ]) {
+    spectrumConfig = {
```

Comment:
We need to add this config to our documentation too. Would you be interested to layout an idea on how we can better communicate this additional config for the users.

Reply:
Also this is not currently static like it in the `Base.ts`, should it be?

Reply:
Yes this should be a static property also
```js
static override spectrumConfig = {
        downstate: ['spectrum-two'],
    };
```